core/jreleaser-model-impl/src/main/java/org/jreleaser/model/internal/deploy/maven/AbstractMavenDeployer.java

```diff
@@ -84,6 +90,7 @@ public void merge(S source) {
         this.username = merge(this.username, source.getUsername());
         this.password = merge(this.password, source.getPassword());
         this.authorization = merge(this.authorization, source.getAuthorization());
+        this.snapshotSupported = false;
```

Comment:
This should be `this.snapshotSupported  = merge(this.snapshotSupported , source.isSnapshotSupported());`